java/ql/src/Metrics/Summaries/GeneratedVsManualCoverage.ql

```diff
@@ -23,12 +23,12 @@ private int getNumMadModeledApis(string package, string provenance) {
       sc.asCallable() instanceof DataFlowTargetApi and
       (
         // "auto-only"
-        sc.isAutoGenerated() and
+        sc.applyGeneratedModel() and
         provenance = "generated"
         or
-        sc.isManual() and
+        sc.applyManualModel() and
         (
-          if sc.hasProvenance(["generated", "ai-generated"])
+          if sc.hasGeneratedModel()
```

Comment:
It looks to me like this update will cause the `GeneratedVsManualCoverage` query to use _all_ generated models, is that right? If so, is there an easy way to specify just the `df-generated` models instead? The query should only take the `df-generated` models into account for now (I've confirmed this with @yo-h).

Reply:
Yes, that is right!
I will change the logic such that only `df-generated` models are considered.
However, this also means that if there exist a callable that has a `manual` and `ai-generated` and not any `df-generated` models, it will be considered `manual` only.

Reply:
> this also means that if there exist a callable that has a manual and ai-generated and not any df-generated models, it will be considered manual only

I think considering the model as manual-only in this situation is fine for now. 

Thanks for updating this!